netcore/startserver.ps1

```diff
@@ -0,0 +1,65 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This looks like part of a test framework maybe, why is it in the root of the subproject?

Reply:
Moved all scripts to `/scripts` dir at root of netcore tree.